### DISCUSSION TITLE
Issues after upgrading the firmware to the June 2023 version

### DISCUSSION THREAD

**Discussion by laffineuse** [thehighnibble/thehighnibble.github.io#108] [IMSAI8080esp replica]
I updated the firmware by dropping the latest version of imsaisim_esp32.bin on SYS: and initially got a "successful" message but now the IMSAI will not reboot.  The lights no longer come on on the front panel.  If I connect the PC directly to the EPSP32, it seems to be caught in a reboot loop.  Where do I go from here?

**Comment by GrantMeStrength** [comment]
Worst case you can unplug the ESP, and give it a damn good reflashing. The instructions and files are in a folder on this GitHub site.

**Comment by nbreeden2** [comment]
Do you know what version you upgraded from. There is a breaking update change discussed here: 

https://github.com/thehighnibble/firmware/releases
and
https://github.com/thehighnibble/firmware/releases/tag/v1.12.0

If you got caught in this change you will need to reflash manually.

**Comment by laffineuse** [comment]
I ended up replacing the content of the MicroSD with the latest image and it worked.Sent from my iPhoneOn Oct 9, 2023, at 17:59, Neil ***@***.***> wrote:﻿
Do you know what version you upgraded from. There is a breaking update change discussed here:
https://github.com/thehighnibble/firmware/releases
and
https://github.com/thehighnibble/firmware/releases/tag/v1.12.0
If you got caught in this change you will need to reflash manually.

**Reply by nbreeden2** [reply]
I believe this was the fix if this was the breaking upgrade issue, but it's been long enough ago for me I don't recall for sure.
